content/blog/living-off-the-ipad-as-an-engineer/index.md

```diff
@@ -0,0 +1,110 @@
+
+The answer is mostly through remote access. Fear not, dear reader, we’ll try to rely on tools that are or *feel* native on the iPad Pro, not just cheap Teamviewer-ing. What started as just a 9 to 5 setup challenge, not the other way around, is now much more than that.
+
+*Note 1: This is merely a shoutout to great products I’m using daily and isn’t sponsored. The links are not affiliated either. I’ll try to provide different options as well as keep some focus on open-source software.*
+
+*Note 2: The new iPad Air now features most of the laptop-like abilities of its Pro brother; therefore, I’ll only use the term “iPad” in the following. But bear in mind: the cheapest 2020 8th-gen iPad still has the old form-factor and a Lightning port, making it incompatible with some of the following.*
+
+![Photo by [Ernest Ojeh](https://unsplash.com/@namzo?utm_source=unsplash&utm_medium=referral&utm_content=creditCopyText) on [Unsplash](https://unsplash.com/s/photos/magic-keyboard?utm_source=unsplash&utm_medium=referral&utm_content=creditCopyText)](cover.jpeg)
+
+## External Monitor
+
+ I was lucky enough to have an [LG Ultrafine 4K](https://www.apple.com/shop/product/HMUA2VC/A/lg-ultrafine-4k-display?fnode=01ed6570fcad8a82974ba2365bf1eaa7d05d2098f328a409870e96ca8209d73125b43501b2178d7e0da5e2a872812d90d869256e6fac4e035ad1db81d91c5dca98f07b0806b261899f5ca87dcdf5f2b3bdd2a569c5a005cfba29a467eb8ce95e6d29ec0c8c4548ba56c842fb8e09c74a) display in my possession for use with the MacBook Pro. These fancy displays designed hand-in-hand with Apple are both compatible with Thunderbolt devices like the MacBook and with standard USB-C devices like the iPad. However, you can’t use the same cable, so if you are to buy this one, make sure you’re using the one with an iPad label on it before returning it out of frustration!
+
+I believe it’s common knowledge that you can’t just work out of a laptop form-factor all day without destroying your neck. Here the cheapest option would be to go with one of the pretty cool arm mount specifically designed to put the iPad right in front of your eyes or just a pile of books. This allows the expensive pixels to get their deserved amount of attention while also enabling instant video calling.
+
+While an external monitor is really comfortable, one thing to note is the lack of *full* external monitor support with iPadOS at the time of writing (version 14.4). Connecting a USB-C display like the Ultrafine essentially triggers an AirPlay mirroring of the iPad’s and is therefore really not as satisfying as a standard laptop+display setup, with the extended desktop for greater multitasking. Depending on its size and resolution, you’ll most likely end up with black bars around the mirrored video flux. Annoying even though you do get used to it. But they are ways around it.
```

Comment:
While an external monitor is really comfortable, one thing to note is the lack of *full* external monitor support with iPadOS at the time of writing (version 14.4). Connecting a USB-C display like the Ultrafine essentially triggers an AirPlay mirroring of the iPad’s screen and is therefore really not as satisfying as a standard laptop+display setup with an extended desktop for greater multitasking. Depending on its size and resolution, you’ll most likely end up with black bars around the mirrored video flux. It's annoying even though you do get used to it. But there are ways around it.